### ISSUE TITLE
Can't register Ddownload.com Premium Account without crash

### ISSUE THREAD

**Issue by 25gGehacktePetersilie** [pyload/pyload#4098]
### Description
Ddownload.com Premium Account cannot be registered without pyload crashing (repeatedly).

### Debug log

administrator@Karen:~$ pyload
Deleting old pidfile /tmp/pyLoad/pyload.pid
[2022-02-22 14:30:55]  INFO                pyload  *** Welcome to pyLoad 0.5.0 ***
[2022-02-22 14:30:55]  INFO                pyload  Activated plugins: ClickNLoad, ExternalScripts, TORRENT, UnSkipOnFail, UserAgentSwitcher
[2022-02-22 14:30:55]  INFO                pyload  Deactivate plugins: AndroidPhoneNotify, AntiCaptcha, AntiStandby, AntiVirus, AppriseNotify, BypassCaptcha, Captcha9Kw, Checksum, CloudFlareDdos, DeathByCaptcha, DeleteFinished, DiscordNotifier, DownloadScheduler, ExpertDecoders, ExtractArchive, HotFolder, IRC, ImageTyperz, JustPremium, LinkFilter, LinkdecrypterComHook, LogMarker, MergeFiles, MultiHome, PushBullet, PushOver, RestartFailed, SkipRev, TransmissionRPC, UpdateManager, WindowsPhoneNotify, XFileSharing, XMPP
[2022-02-22 14:30:55]  INFO                pyload  User directory: /home/administrator/.pyload
[2022-02-22 14:30:55]  INFO                pyload  Cache directory: /tmp/pyLoad
[2022-02-22 14:30:55]  INFO                pyload  Storage directory: /home/administrator/Downloads/pyLoad
[2022-02-22 14:30:55]  INFO                pyload  Storage free space: 1.15 TiB
[2022-02-22 14:30:55]  INFO                pyload  Activating accounts...
[2022-02-22 14:30:55]  INFO                pyload  Activating plugins...
[2022-02-22 14:30:55]  WARNING             pyload  ADDON TORRENT: torrents / magnets are not associated with any plugin
[2022-02-22 14:30:55]  INFO          pyload.webui  Starting webserver: http://localhost:8000
 * Serving Flask app "pyload.webui.app" (lazy loading)
 * Environment: development
 * Debug mode: on
 * Running on http://localhost:8000/ (Press CTRL+C to quit)
127.0.0.1 - - [22/Feb/2022 14:30:57] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:30:57] "POST /json/status HTTP/1.1" 200 -
[2022-02-22 14:31:00]  INFO                pyload  ADDON ClickNLoad: Proxy listening on 0.0.0.0:9666
127.0.0.1 - - [22/Feb/2022 14:31:01] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:01] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:05] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:05] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:09] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:09] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:13] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:13] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:17] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:17] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:21] "POST /json/status HTTP/1.1" 200 -
127.0.0.1 - - [22/Feb/2022 14:31:21] "POST /json/status HTTP/1.1" 200 -
[2022-02-22 14:31:21]  INFO                pyload  ACCOUNT DdownloadCom: Adding user `ddo*******`...
[2022-02-22 14:31:21]  INFO                pyload  ACCOUNT DdownloadCom: Login user `ddo****`...
Speicherzugriffsfehler (Speicherabzug geschrieben)
administrator@Karen:~$

### Additional references

With other premium accounts I do not have this issue.

"Speicherzugriffsfehler (Speicherabzug geschrieben)" is german and means "Memory error (memory dump written)".

**Comment by GammaC0de** [comment]
I don't know if it would help but please disable `Development mode` of pyload's "Web Interface" settings

**Comment by 25gGehacktePetersilie** [comment]
No, unfortunately this does not help i.e. pyload still crashes...

**Comment by GammaC0de** [comment]
This errors comes from your system or maybe from buggy pycurl implemantation
Are you using some kind of VPN?

**Comment by 25gGehacktePetersilie** [comment]
No VPN is running but in my router I have activated DNS over TLS (DoT) however pyload also crashes with deactivated DoT. I reinstalled pycurl (python3-pycurl from Ubuntu 20.04 repository) but also this is not helping :-(

**Comment by DerEmder** [comment]
Same here, same error messages (Speicherzugriffsfehler). DDL.to is not working. Please fix.

**Comment by uysct** [comment]
Unfortunately I have to confirm this problem. As a premium user, I can download a maximum of three files. Then no more traffic is shown available in pyload and pyload crashed at the same time. After the restart I have to add the ddownload account again. Is there a solution to this at the moment?

**Comment by GammaC0de** [comment]
@uysct Please go  to Settings -> Web Interface 
and disable "Development mode", restart pyload and retry

**Comment by uysct** [comment]
@GammaC0de Thanks for the feedback. Unfortunately, your suggestion doesn't solve the problem. I also forgot to tell that I'm still using the normal stable pyload 0.4.20. The following messages appear in the debug `log:

ACCOUNT DdownloadCom: VALID UNTIL PATTERN not found
ACCOUNT DdownloadCom: TRAFFIC LEFT PATTERN not found
ACCOUNT DdownloadCom: LEECH TRAFFIC PATTERN not found

`

**Comment by GammaC0de** [comment]
This was fixed on the recent plug-in update (from yesterday)

Why not switch to v0.5?
Maybe it would solve your issue..

**Comment by uysct** [comment]
Thank you! 🙂
